### DISCUSSION TITLE
Can SimNIBS match the electrode position with the cortical activation position?

### DISCUSSION THREAD

**Discussion by Jaycob-jh** [simnibs/simnibs#278] [9. Results and analysis]
Dear development team,
 I‘m not very familiar with SimNIBS yet. From the location of neural activity inside the brain (such as the brain Atlas module), how should I correspond to the external brain epidermal stimulation location of tES? In other words, how to solve the inverse problem, can SimNIBS match the electrode position with the cortical activation position?

**Comment by TorgeW** [comment]
You can use our TDCS optimization:
https://simnibs.github.io/simnibs/build/html/tutorial/optimization.html

**Comment by Jaycob-jh** [comment]
thank you so much and it can solve some problems.Does this "project on scalp" module have any specific algorithms or literature to support how it does this projection?

**Reply by TorgeW** [reply]
Its using the closest point on the scalp surface.
